### ISSUE TITLE
Preferences file location?

### ISSUE THREAD

**Issue by jessemillar** [Kalejin/DCSB#63]
I'd like to back up my settings. Where is the preferences file(s) saved? I can't easily find it in Explorer or browsing the code (I'm likely missing something obvious).

**Comment by AlberLC** [comment]
Yeah! I would love to have access to a configuration file too! So I could use external tools to create my presets directly from my folders with sounds or to save my presets without having to copy the entire program.

Please, could you move forward if this option is to be developed?

Good job for this great application.
